### PR TITLE
Add IP Space floating IP suggestion method

### PR DESCRIPTION
**Pull request by Didainius**
This PR adds `*VCDClient.GetAllIpSpaceFloatingIpSuggestions` method as well as relevant type `IpSpaceFloatingIpSuggestion`

### CODE REVIEW

**Review by rocknes — APPROVED** [review]
:lgtm:

File: govcd/ip_space_allocation.go [hunk 1]
```diff
@@ -279,3 +281,30 @@ func getAllIpSpaceAllocations(client *Client, ipSpaceId string, org *Org, queryP
 
 	return results, nil
 }
+
+// GetAllIpSpaceFloatingIpSuggestions suggests IP addresses to use for networking services on Edge
+// Gateway or Provider Gateway. 'gatewayId' is mandatory. Based on the specified Gateway, VCD
+// will query all the applicable IP Spaces and suggest some IP addresses which can be utilized to
+// configure the network services on the Gateway. IP Space IP addresses which are are allocated but
+// not currently used for any network services are considered. Results can also be filtered by IPV4
```
Comment (lvirbalas) [inline govcd/ip_space_allocation.go:289]: A typo. I'm also thinking how to make this description feel less "floaty" (i.e. make it more determined) :)

Something like this:

```suggestion
// configure the network services on the Gateway. Allocated IP Space's IP addresses, but
// not currently used for any network services are returned. Results can also be filtered by IPV4
```

**Review by lvirbalas — APPROVED** [review]
One ask and looks great!

**Review by dataclouder — APPROVED** [review]

File: govcd/ip_space_allocation.go (hunk 1 above)
Comment (Didainius) [reply govcd/ip_space_allocation.go]: I have set it to be such. This comes from API docs, but indeed might sound confusing.

**Review by adambarreiro — APPROVED** [review]
Thanks!

**Pull request merged**